php/mailchimp.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+class MailChimpException extends Exception { };
+
+/**
+ * Class MailChimp
+ */
+
+class MailChimp {
+
+	private $api_uri;
+	private $api_user = 'api_v3';
```

Comment:
Considering this value should never change at run time, it should probably be a class constant:
```
const API_USER = 'api_v3';
```
Access with:
```
self::API_USER
```

*Reference*:
- http://php.net/manual/en/language.oop5.constants.php